src/services/eth.js

```diff
@@ -13,13 +14,16 @@ export default class Ethereum {
     const providerUrl = process.env.ETHEREUM_RPC_URL
     this.provider = new ethers.providers.JsonRpcProvider(providerUrl)
     this.network = network
-
+    this.spenders = {
+      balancer: process.env.EXCHANGE_PROXY,
+      uniswap: process.env.UNISWAP_ROUTER
+    }
+    this.tokenListUrl = process.env.ETHEREUM_TOKEN_LIST_URL
     if (network === 'kovan') {
```

Comment:
Names of networks should be stored on one file, it's can be enum collection, in case of using Typescript you can use enum items in code. In case of using Javascript you can use simply constants.